test/testcafe/framework/page-objects/DeviceCodeActivatePageObject.js

```diff
@@ -72,14 +71,14 @@ export default class DeviceCodeActivatePageObject extends BasePageObject {
 
   isTerminalSuccessIconPresent() {
     if (userVariables.gen3) {
-      return this.form.elementExist('.infobox-success');
+      return this.form.getAlertBox().hasClass('MuiAlert-calloutSuccess');
```

Comment:
I believe the `data-se="message"` on the top level `Box` that wraps the `Callout` component is not actually used in our page objects (I did a CMD+F for [data-se="message"]). If that's so, we could replace that `data-se` with the ``infobox-${messageClass.toLowerCase()}`` and then use that for all selectors where we need the callout variant

Reply:
Done
https://github.com/okta/okta-signin-widget/pull/3484/commits/11c32491a5bdbde59e9f375e6bda4c5fd31587f5#diff-3c0545bd20e65728cb0870b895809baafc3b5da0137f9d83d738ee775f687c8dR46